backend/src/apiserver/main.go

```diff
@@ -106,12 +106,23 @@ func main() {
 	}
 	log.SetLevel(level)
 
+	go reconcileSwfCrs(resourceManager)
+
 	go startRpcServer(resourceManager)
 	startHttpProxy(resourceManager)
 
 	clientManager.Close()
 }
 
+func reconcileSwfCrs(resourceManager *resource.ResourceManager) {
+	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Minute)
+	defer cancel()
+	err := resourceManager.ReconcileSwfCrs(ctx)
+	if err != nil {
+		log.Errorf("Could not reconcile the ScheduledWorkflow Kubernetes resources: %v", err)
+	}
+}
```

Comment:
I know the rest of the code doesn't do this for separate go routines but I think the correct approach would be something like this to properly wait for the goroutine to finish.
```suggestion
	backgroundCtx, backgroundCancel := context.WithCancel(context.Background())
	defer backgroundCancel()
	wg := sync.WaitGroup{}
	wg.Add(1)
	go reconcileSwfCrs(resourceManager, ctx, &wg)

	go startRpcServer(resourceManager)
	// This is blocking
	startHttpProxy(resourceManager)

	backgroundCancel()
	clientManager.Close()
	wg.Wait()
}

func reconcileSwfCrs(resourceManager *resource.ResourceManager, ctx context.Context, wg *sync.WaitGroup) {
	defer wg.Done()
	err := resourceManager.ReconcileSwfCrs(ctx)
	if err != nil {
		log.Errorf("Could not reconcile the ScheduledWorkflow Kubernetes resources: %v", err)
	}
}
```

Reply:
Done.